### DISCUSSION TITLE
Charts export

### DISCUSSION THREAD

**Discussion by ZML0113** [Open-Systems-Pharmacology/Forum#1403] [Questions & Problems] [answered]
<img width="381" alt="1669859024959" src="https://user-images.githubusercontent.com/81542635/204945915-f5b9b171-e6fe-4272-94ab-f9e32d921d73.png">
When I try exporting the chart to clipboard, it has a watermark. I would like to know how to remove it.

**Comment by msevestre** [comment, marked as answer]
Go to your user settings and deselect the watermark option
